addons/rose/index.js

```diff
@@ -30,6 +30,10 @@ module.exports = {
         destDir: this.name,
         include: ['**/*'],
       }),
+      new Funnel(stylePath, {
```

Comment:
How does this work, does the monokai theme need to be imported too? It  doesn't look like it worked for me:
<img width="1502" alt="image" src="https://user-images.githubusercontent.com/5783847/184904387-44717448-e804-4361-9262-0a1f8a849f14.png">